src/BitStringAddresses/fermifs.jl

```diff
@@ -131,6 +131,54 @@ Base.:(==)(a::FermiFS, b::FermiFS) = a.bs == b.bs
 num_occupied_modes(::FermiFS{N}) where {N} = N
 occupied_modes(a::FermiFS{N,<:Any,S}) where {N,S} = FermiOccupiedModes{N,S}(a.bs)
 
+num_unoccupied_modes(::FermiFS{N,M}) where {N,M} = M - N
+function unoccupied_modes(a::FermiFS{N,M,S}) where {N,M,S}
+    s = ~a.bs
+    FermiUnoccupiedModes{M - N,typeof(s)}(s)
+end
```

Comment:
This should specialise for the storage type `S` and only invert the bitstring with `~` if the storage type is a bitstring.